src/client/debugger/jupyter/debuggingManager.ts

```diff
@@ -304,7 +304,10 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDeb
             __mode: mode,
             __cellIndex: cell.index
         };
-        const opts = mode === KernelDebugMode.RunByLine ? { debugUI: { simple: true } } : undefined;
+        const opts: DebugSessionOptions | undefined =
+            mode === KernelDebugMode.RunByLine
+                ? { debugUI: { simple: true }, suppressSaveBeforeStart: true }
```

Comment:
@roblourens
Is this avaialble in VS Code 1.60 stable?

Reply:
No

Reply:
But it will be ignored and not cause an issue, if that's what you're wondering